disease-classification-training/unittests.py

```diff
@@ -21,13 +21,17 @@ def setUp(self) -> None:
         self.disease_classification = train_model(
             url=self.test_cfg["lr"],
             token=self.test_cfg["epochs"],
-            org=self.test_cfg["batch_size"],
-            org=self.test_cfg["model_name"]
+            bs=self.test_cfg["batch_size"],
+            model_name=self.test_cfg["model_name"]
+            train_data_size=sef.test_cfg["test_num"]
```

Comment:
looks like a typo here: sef.test_cfg and is 'test_num' parameter defined in cfg file?

Reply:
This was a typo and now it changed to 'train_data_size'